docs/developer/how-to/make-new-ophyd-async-device.md

```diff
@@ -0,0 +1,58 @@
+# Decision Flowchart for Creating a New ophyd_async Device
+
+This document contains decision flowcharts designed to guide developers through the process of creating a new ophyd_async device in the Ophyd library. These flowcharts help in determining the appropriate class inheritance, methods to override, and testing procedures for optimal device functionality and integration into the system.
```

Comment:
Must: Dodal:

```suggestion
This document contains decision flowcharts designed to guide developers through the process of creating a new ophyd_async device in the dodal library. These flowcharts help in determining the appropriate class inheritance, methods to override, and testing procedures for optimal device functionality and integration into the system.
```

Could: Also, this sentence suggests that the device doesn't have to be ophyd-async. I think we're fairly settled on that as a _requirement_ for dodal at this point so feel free to say that.

Reply:
The alternative is ophyd. It doesn't strongly suggest it, I just think it might be worth saying something like: "ophyd-async is the standard hardware abstraction library for Diamond, old-style ophyd devices should no longer be used."

Reply:
who even knows about the old 'ophyd' at this point? was it used live more than once?

I don't see any source of misleading someone *towards* thinking 'I'll use old ophyd' that would warrant the disclaimer here.